src/Selenium2Driver.php

```diff
@@ -656,12 +656,18 @@ public function setValue($xpath, $value)
         $value = strval($value);
 
         if (in_array($elementName, array('input', 'textarea'))) {
-            $existingValueLength = strlen($element->attribute('value'));
+            $deleteSequence = Key::CONTROL . 'a' . KEY::NULL_KEY;
```

Comment:
On Mac the shortcut for doing this is Command+A. We should detect OS and use corresponding shortcut.